tweet_recommendations/other_methods/graph_summarization_for_hashtag_recommendation.py

```diff
@@ -10,11 +10,12 @@
 from sklearn.feature_extraction.text import TfidfVectorizer
 
 from tweet_recommendations.utils.clients import get_wcrft2_results_for_text
-from .method import Method
+from estimator import Estimator
```

Comment:
Ten import się popsuł chyba

Reply:
Potwierdzam, że jest niepoprawny. Mimo wszystko działa przy odpaleniu validatora odpowiedniego, co mnie dziwi. @jmichalowski12 poprawisz przy okazji?

Reply:
yup, przez przypadek się to chyba stało :(

Reply:
@jmichalowski12 to pewnie przez mój popsuty refactor :(